src/control/controlobjectscript.h

```diff
@@ -36,9 +37,10 @@ class ControlObjectScript : public ControlProxy {
 
   protected slots:
     // Receives the value from the master control by a unique queued connection
-    void slotValueChanged(double v, QObject*);
+    virtual void slotValueChanged(double v, QObject*);
```

Comment:
Why virtual? I don't see any override?

Reply:
This is required for the unit test gmock

Reply:
Then please add a comment

Reply:
Comment added!